internal/client/grpc/things/thingsclient.go

```diff
@@ -0,0 +1,40 @@
+package things
+
+import (
+	"io"
+	"io/ioutil"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+	gogrpc "google.golang.org/grpc"
+
+	"github.com/mainflux/mainflux"
+	thingsapi "github.com/mainflux/mainflux/things/api/auth/grpc"
+
```

Comment:
Rename file to `client.go`.

Reply:
done in both interna/client/grpc/auth and internal/client/grpc/thing